data/academic_institutions/pennsylvania.md

```diff
@@ -7,17 +7,22 @@ logo: academic_institution/penn.png
 continent: North America
 courses:
     - name: Compilers
-      professor: Steve Zdancewic
+      teacher: Steve Zdancewic
       last_check: "2024-03-06"
       acronym: CIS 3410
       enrollment: "70"
-      online_resource: "https://www.seas.upenn.edu/~cis3410/current/"
+      url: "https://www.seas.upenn.edu/~cis3410/current/"
+      exercises: true
+      description: |
+        The provided homework projects of this compilers class progress from basic OCaml to building an X86lite simulator and an LLVMlite backend. Later assignments focus on developing and extending the Oat language compiler, as well as program analysis and optimizations.
```

Comment:
I'm not sure what to do with these @sabine. Words like "optimization" has an "s" in UK spelling, but this is a US class. My opinion is to stay consistent with the UK spelling in descriptions, but if Optimization is in the title of the course, keep the US spelling. What do you think?

Reply:
makes sense to me